lib/routes/ieee/journal.ts

```diff
@@ -31,17 +34,37 @@ async function handler(ctx) {
     const list = tocData.records.map((item) => {
         const mappedItem = mapRecordToItem(volume)(item);
 
-        mappedItem.description = art(path.join(__dirname, 'templates/description.art'), {
-            item: mappedItem,
-        });
-
         return mappedItem;
     });
 
+    const items = await Promise.all(
+        list.map((item) =>
+            cache.tryGet(item.link, async () => {
+                const response = await ofetch(`https://ieeexplore.ieee.org${item.link}`);
+
+                const $ = load(response);
+
+                const target = $('script[type="text/javascript"]')
+                    .filter((_, el) => /xplGlobal\.document\.metadata\s*=/.test($(el).html() || ''))
+                    .first();
```

Comment:
```suggestion
                const target = $('script[type="text/javascript"]:contains("xplGlobal.document.metadata")');
```
`filter` will keep going even after it matches.